src/components/header/header-map.tsx

```diff
@@ -0,0 +1,9 @@
+import { headerData } from './header-data';
+import HeaderEnum from './header-enum';
+
+export const pathToEnumMap: Record<string, HeaderEnum> = {};
+
+headerData.forEach((tab) => {
```

Comment:
Make this a function.

Add UT.

Reply:
I've previously written a UT that tests pathToEnumMap to match tab hrefs to header enum, would that be sufficient?